tests/storage/repository/filesystem/config/test_config_files.py

```diff
@@ -124,7 +131,16 @@ def test_parse_outputs(tmp_path: Path) -> None:
             )
         },
     )
-    assert ConfigPathBuilder.build(study_path, "id") == config
+    assert build(study_path, "id") == config
+
+
+def test_parse_outputs_discriminate_zips(tmp_path: Path) -> None:
+    path_study = ASSET_DIR / "test_output_zip_not_zipped.zip"
+    with ZipFile(path_study) as zip_output:
+        zip_output.extractall(path=tmp_path)
+    output_path = tmp_path / "test_output_zip_not_zipped" / "output"
+    output_parsed = _parse_outputs(output_path)
+    assert not output_parsed["20230127-1550eco"].archived
```

Comment:
Je te suggère d'utiliser des [`tests paramétrés`](https://docs.pytest.org/en/6.2.x/parametrize.html) afin de gérer plus de cas.

Voici le même code en utilisant `@pytest.mark.parametrize`:

```python
import pytest

...

@pytest.mark.parametrize(
    "assets_name, expected",
    [
        (
            "test_output_zip_not_zipped.zip",
            {
                "20230127-1550eco": Simulation(
                    name="",
                    date="20230127-1550",
                    mode="economy",
                    nbyears=1,
                    synthesis=True,
                    by_year=False,
                    error=False,
                    playlist=[],
                    archived=False,
                    xpansion=False,
                )
            },
        ),
    ],
)
def test_parse_outputs__nominal(
    tmp_path: Path, assets_name: str, expected: dict
) -> None:
    """
    This test decompresses a zipped study (stored in the `assets` directory)
    into a temporary directory and executes the parsing of the outputs.
    The result of the analysis is checked to match the expected output data.
    """
    pkg_dir = ASSETS_DIR.joinpath(assets_name)
    with ZipFile(pkg_dir) as zf:
        zf.extractall(tmp_path)
    study_dir = tmp_path.joinpath(assets_name).with_suffix("")
    output_path = study_dir.joinpath("output")
    actual = _parse_outputs(output_path)
    assert actual == expected
```

Il faudrait ajouter des cas de test (des ZIPs) pour les situations suivantes :

- Étude avec un dossier `output` sans sous-dossier `about-the-study` pour vérifier l'absence de données de simulation, autrement dit : `expected = {}`
- Étude avec un dossier `output/foo` sans le fichier `checkIntegrity.txt`,
- Étude avec un dossier `output/foo` avec le fichier `lp/lp_namer.log`,
- Étude avec une archive `output/foo.zip` sans le fichier `checkIntegrity.txt`,
- Étude avec une archive `output/foo.zip` avec le fichier `lp/lp_namer.log`,

À chaque fois, il faudrait ajouter un ZIP dans `tests/storage/business/assets`. Je pense qu'on a pas besoin de toute l'arborescence, donc tu peux éviter d'y mettre `input`, `layers`, etc. et ne garder que `output`. Autrement dit : ne garder que ce qui est réellement nécessaire aux cas de tests.